routes/quiz.js

```diff
@@ -9,11 +9,33 @@ var http = require("http");
 var players_model = require('../models/players.js');
 var leaders_model = require('../models/leaders.js');
 var leaguesearch = require('./leaguesearch.js');
+var teams_model = require('../models/teams.js');
+
     
 
 router.get('/', function(req, res) {
   res.quiz_page = true;
   var quiz_id = req.query.id;
+  var tId = req.query.team_id;
+
+  if (typeof tId != 'undefined' && typeof quiz_id == 'undefined'){
+    //we've only been given a team id. Look up the needed and info and create a quiz from there.
+    db.collection('teams').findOne( { team_id : tId}, function (err, item){
+         if (players_model.goatsLeadersArray().indexOf(item.type) > -1){
+          var api_team_id = null;
+          var quiz_name = item.category;
+        }
+        else{
+          var api_team_id = item.api_team_id;
+          var quiz_name = item.market + ' ' + item.name;
+        }
+        teams_model.createQuiz(tId, item.league, quiz_name, res, redirectQuiz, api_team_id, item.type);
```

Comment:
this part is kinda neat tho. We are passing createQuiz a different callback than the normal one which will in turn map us to the right page